.htaccess

```diff
@@ -100,6 +100,7 @@ RewriteRule ^form[\.:](ipc|IPC)$ enter_bug.cgi?product=Marketing&format=ipc [QSA
 RewriteRule ^form[\.:]nda$ enter_bug.cgi?product=Legal&format=nda [QSA]
 RewriteRule ^form[\.:]name[\.:]clearance$ enter_bug.cgi?product=Legal&format=name-clearance [QSA]
 RewriteRule ^form[\.:]shield[\.:]studies$ enter_bug.cgi?product=Shield&format=shield-studies [QSA]
+RewriteRule ^form[\.:]firefox[\.:]security$ enter_bug.cgi?product=Firefox&format=firefox-security  [QSA]
```

Comment:
could we change /form.firefox.security to /form.client.bounty ? This would be the form for Focus and other non-Firefox client things too.